packages/admin/resources/views/components/layouts/base.blade.php

```diff
@@ -13,6 +13,10 @@ class="filament antialiased bg-gray-100 js-focus-visible"
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <meta name="csrf-token" content="{{ csrf_token() }}">
 
+        @foreach (\Filament\Facades\Filament::getMeta() as $tag)
+            {{ new \Illuminate\Support\HtmlString($tag) }}
```

Comment:
We shouldn't wrap the `$tag` in a `HtmlString` - we should just output `$tag` normally and let the user decide if they want to pass an `HtmlString` or not.

Reply:
I can agree with this. My thought was that there shouldn't ever be a scenario where it wouldn't be an HTML tag since it's in the head, but perhaps this just needs to be documented.

I'll update it and we can go from there!